CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
 
+## 1.1.1
```

Comment:
I suggest calling the upcoming version "vNext" until we define what version we're releasing.